libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -468,6 +468,15 @@ type AccountAPI =
                :> ReqBody '[JSON] NewPasswordReset
                :> MultiVerb 'POST '[JSON] '[RespondEmpty 201 "Password reset code created and sent by email."] ()
            )
+    :<|> Named
+           "post-password-reset-complete"
+           ( Summary "Complete a password reset."
+               :> CanThrow 'InvalidPwResetCode
+               :> "password-reset"
+               :> "complete"
+               :> ReqBody '[JSON] CompletePasswordReset
+               :> MultiVerb 'POST '[JSON] '[RespondEmpty 200 "Password reset successful."] ()
+           )
```

Comment:
I didn't understand that you meant `CanThrow` not being type checked in brig when you mentioned it in standup. IIRC, I was using `MultiVerb` and `FromUnion`, `ToUnion` classes to ensure things were type checked.